tests/resource_test.js

```diff
@@ -0,0 +1,65 @@
+/// <reference path="./steps.d.ts" />
+Feature('Resource test');
+
+var FirstLogin = false;
+var loginCookies = [];
+const pdfName = `Dummy PDF ${Math.random() * 1000}`;
+const collectionName = `Dummy Collection ${Math.random() * 1000}`;
+
+Before((I) => {
+    I.login('admin', 'password');
```

Comment:
Note to self: we should probably make some global variable for login stuff, so if we ever change the default we don't have to edit every test.